src/code/z_quake.c

```diff
@@ -1,7 +1,8 @@
-#include "prevent_bss_reordering.h"
-#include "global.h"
-#include "libc/string.h"
 #include "z64quake.h"
+
+#include "libc/string.h"
+
+#include "global.h"
 #include "z64view.h"
```

Comment:
I think I see the logic behind these reorders
```
file's header

libc/otherlibs?

other headers
```

We should probably write it down somewhere